tests/integration/s3/test_key.py

```diff
@@ -419,7 +419,10 @@ def test_header_encoding(self):
         remote_metadata = check._get_remote_metadata()
 
         # TODO: investigate whether encoding ' ' as '%20' makes sense
-        self.assertEqual(check.cache_control, 'public,%20max-age=500')
+        self.assertIn(
+            check.cache_control,
+            ('public,%20max-age=500', 'public, max-age=500')
+        )
```

Comment:
Yep - didn't mention it in the commit message, but I put a little note about it at the bottom of the pull request.  It's just been an annoying test "failure" to have to look over every time I run the s3 integration tests.